docs/production_setup.rst

```diff
@@ -125,7 +125,7 @@ are run within the context of a specific settings file, discussed shortly.
 
   python manage.py refresh
   python manage.py collectstatic --noinput
-  waitress-serve --port=$VCAP_APP_PORT atf_eregs.wsgi:application
+  gunicorn atf_eregs.wsgi:application
```

Comment:
Gunicorn checks for `$PORT` automatically, and CF provides it (`VCAP_APP_PORT` is deprecated). Is that too magical? We can be more explicit and say `-b 127.0.0.1:$PORT`.